<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTasks</class>
 <widget class="QDialog" name="AddTasks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>14</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Add Task</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="ql">
     <item row="7" column="1">
      <widget class="QLineEdit" name="le_asnwer3">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>&amp;Right Key</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>3.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="le_answer1">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>1.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="le_answer2">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>&amp;Answers</string>
       </property>
       <property name="buddy">
        <cstring>le_answer1</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>2.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_task">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="l_task">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>&amp;Enter task</string>
       </property>
       <property name="buddy">
        <cstring>le_task</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_finish">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>&amp;Finish</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_nexttask">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>17</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>&amp;Next task</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_answer1</tabstop>
  <tabstop>le_answer2</tabstop>
  <tabstop>le_asnwer3</tabstop>
  <tabstop>pb_cancel</tabstop>
  <tabstop>pb_finish</tabstop>
  <tabstop>pb_nexttask</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>AddTasks</sender>
   <signal>rejected()</signal>
   <receiver>AddTasks</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
